### DISCUSSION TITLE
Select component dont display selected values

### DISCUSSION THREAD

**Discussion by KevinEscobarV** [wireui/wireui#278] [BUG] [answered]
I noticed that if the data inside the collection is of type string it works

doesn't work if:
`public $model = [1, 2, 3];` ❌
but it works if:
`public $model = ['1', '2', '3'];` ✅

In my case my model is:
`public $model;`
`$this->model = $user->diseases->pluck('id')->toArray();`

I can't find a solution

i am using multiselect, in my edit modal, but dont display selected values:

```
<x-select label="Select a disease" placeholder="Select an option"
                    wire:model.defer="model" multiselect>
                    @foreach ($diseases as $disease)
                        <x-select.option label="{{ $disease->name}}" value="{{ $disease->id }}" />
                    @endforeach
</x-select>
```

**Comment by ph7jack** [comment, marked as answer]
@KevinEscobarV 
Could you try to pass the id as a number?
The WireUI Select uses a strict comparison. The value "1" is not equal to 1
You can pass a number using :value="1"

```
<x-select label="Select a disease" placeholder="Select an option"
                    wire:model.defer="model" multiselect>
                    @foreach ($diseases as $disease)
                        <x-select.option :label="$disease->name" :value="$disease->id" />
                    @endforeach
</x-select>
```
